test/middleware/test-express.ts

```diff
@@ -116,15 +113,14 @@ describe('middleware/express', () => {
     assert.strictEqual(passedProjectId, FAKE_PROJECT_ID);
   });
 
-  [envDetect.GCPEnv.APP_ENGINE, envDetect.GCPEnv.CLOUD_FUNCTIONS].forEach(
-      env => {
-        it(`should not generate the request logger on ${env}`, async () => {
-          authEnvironment = env;
-          await middleware();
-          assert.ok(passedOptions);
-          assert.strictEqual(passedOptions.length, 1);
-          // emitRequestLog parameter to makeChildLogger should be undefined.
-          assert.strictEqual(passedEmitRequestLog, undefined);
-        });
-      });
+  [GCPEnv.APP_ENGINE, GCPEnv.CLOUD_FUNCTIONS].forEach(env => {
```

Comment:
So not directly related to this PR - but I typically prefer to just copy the code for tests like this.  Debugging a failure is hard because the stack trace becomes a guess, and things like `.only` and `.skip` become harder.